### PR TITLE
Estimates initial parameters for exponential function in Iqtfit

### LINKED ISSUE #27200: IndirectDataAnalysis IqtFit StretchExp calculates start values from the data. Exp(1) should also do it.
In the Iqt fit tab there are two possible fitting functions which can be added. An exponential and a stretched exponential. Of these the stretched exponential is automatically populated with reasonable values whereas the exponential is not. This should be updated so they are both populated with reasonable values.

### PR DESCRIPTION
**Pull request by RichardWaiteSTFC**
**Description of work.**
Estimate initial parameters for the exponential function(s) in Iqtfit as was implemented already for the stretched exponential. 

**To test:**

1. Load data (e.g. script below)

from mantid.simpleapi import *

Load(Filename='irs26174_graphite002_red.nxs', OutputWorkspace='irs26174_graphite002_red')
Load(Filename='irs26173_graphite002_res.nxs', OutputWorkspace='irs26173_graphite002_res') 
TransformToIqt(SampleWorkspace='irs26174_graphite002_red', ResolutionWorkspace='irs26173_graphite002_res', 
    ParameterWorkspace='irs26174_graphite002_TransformToIqtParameters', OutputWorkspace='irs26174_graphite002_iqt')

2. Open the Indirect >Data Analysis interface
3. Go to the I(Q,t) Fit
4. Load sample from workspace (select only valid workspace)
5. Add 1 and 2 exponential functions and check that the initial parameters are not (Height=1, Lifetime=1) and are different for a few spectra. The height of the second exp should be 10% of the first and the lifetime should be double (arbitrary choice of second exp initial parameters)

image

Fixes #27200 

This does not require release notes because it is a small change.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by AnthonyLim23** [comment]
@Mantid-Matthew could you test this?

**Review by DannyHindson — CHANGES_REQUESTED** [review]
Tested in workbench. Initial estimates are now set rather than just being 1. I've added a few detail level comments

File: qt/scientific_interfaces/Indirect/IndirectFunctionBrowser/IqtFunctionModel.cpp [hunk 1]
```diff
@@ -625,6 +628,34 @@ void IqtFunctionModel::estimateStretchExpParameters() {
   }
 }
 
+void IqtFunctionModel::estimateExpParameters() {
+  auto const heightName1 = getParameterName(ParamID::EXP1_HEIGHT);
+  auto const lifeTimeName1 = getParameterName(ParamID::EXP1_LIFETIME);
+  if (!heightName1 || !lifeTimeName1)
+    return;
+  auto const heightName2 = getParameterName(ParamID::EXP2_HEIGHT);
+  auto const lifeTimeName2 = getParameterName(ParamID::EXP2_LIFETIME);
```
Comment (DannyHindson) [inline qt/scientific_interfaces/Indirect/IndirectFunctionBrowser/IqtFunctionModel.cpp]: The check on L634 to see if heightName1 or lifeTimeName1 are empty needs extending to include the xxx2 variables as well. So you just need to move the if statement down a couple of lines and extend it

**Review by DannyHindson — COMMENTED** [review]
Tested it again and it works. I think there are two minor changes still needed - see new comments

**Review by DannyHindson — DISMISSED** [review]
All looks good now thanks. Tested the I(Q,t) fit again in the Indirect Data Analysis Screen in MantidPlot and the initial estimates are as required

**Comment by NickDraper** [comment]
I feel this needs something in the release notes

**Review by NickDraper — DISMISSED** [review]

**Review by NickDraper — APPROVED** [review]
builds cancelled to preserve resources

**Pull request merged**